### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by ryekerjh**
@ayojackay Please review

### CODE REVIEW

**Comment by ayojackay** [comment]
Nice!

File: index.html [hunk 1]
```diff
@@ -8,10 +8,35 @@
...
+		        <span class="icon-bar"></span>
+		        <span class="icon-bar"></span>
+		      </button> <!--Hamburger menu ends -->
+		      <a class="navbar-brand" href="#"><img src="img/name-01.png" alt="Restaurante"></a>
+		    </div>
+
+		    <!-- Collect the nav links, forms, and other content for toggling -->
+		    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+		      <ul class="nav navbar-nav navbar-right">
+						<li class="active"><a href="#">Home <span class="sr-only">(current)</span></a></li>
```
Comment (ryekerjh) [inline index.html:29]: Too much space, please remove extra whitespace

**Comment by ayojackay** [comment]
Okay, after reviewing your code. no issues. will merge

**Review by ryekerjh — COMMENTED** [review]
Add the Fresno Bee Styles back into the css file

**Pull request merged**